src/event-store/health/event-store.health-indicator.ts

```diff
@@ -0,0 +1,30 @@
+import { HealthIndicator, HealthIndicatorResult } from '@nestjs/terminus';
+import { Inject, Injectable } from '@nestjs/common';
+import { EVENT_STORE_CONNECTOR } from '../services/event-store.constants';
+import { Client } from '@eventstore/db-client/dist/Client';
+
+@Injectable()
+export class EventStoreHealthIndicator extends HealthIndicator {
+  constructor(
+    @Inject(EVENT_STORE_CONNECTOR)
+    private readonly eventStore: Client,
+  ) {
+    super();
+  }
+
+  public check(): HealthIndicatorResult {
+    // if (!this.eventStore.isConnected()) {
```

Comment:
There is a reason to leave commented code?

Reply:
yes, I'm actually updating the health indicator system